app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb

```diff
@@ -9,8 +9,8 @@
 
         <%= render Content::ChecklistCollageComponent.new(
           checklist: [
-            "You'll earn at least £30k (or more in London) if you're just starting as a teacher.",
-            "The average salary for a classroom teacher in England is over £40k a year."
+            "You'll earn at least #{v :salaries_starting_minshortened } (or more in London) if you're just starting as a teacher.",
```

Comment:
```suggestion
            "You'll earn at least <%= v :salaries_starting_minshortened %> (or more in London) if you're just starting as a teacher.",
```

Reply:
@Sarah-DfE I was wrong about this - it didn't a `<%=` at the format as there is already this on line 10

I am cancelling this suggestion - and @martyn-w is looking at the linting error

@martyn-w Please can you update the guidance documentation as well to cover this scenario?